.changeset/good-rocks-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/i18n": patch
```

Comment:
Patch console and myaccount as well.
- For i18n change to be reflected in console, console patch is required.
- Since myaccount depends on i18n, let's bump that as well